docs/tutorial/part-five/index.md

```diff
@@ -91,7 +91,7 @@ Space</kbd>).
 
 ![filesystem-autocomplete](filesystem-autocomplete.png)
 
-Try adding a number of fields to your query, pressing <kbd>Ctrl + Enter</kbd>
+Try adding a number of fields to your query pressing <kbd>Ctrl + Enter</kbd>
```

Comment:
The comma seems correct in this instance?

Reply:
Oh, I must have removed it by accident and didn't catch it later.

Reply:
No worries!